config/district_somerville.yml

```diff
@@ -0,0 +1,74 @@
+config:
+  remote_filenames:
+    FILENAME_FOR_STUDENTS_IMPORT: >-
+      students_export.txt
```

Comment:
What does the `>-` do here?  I was expecting a plain

```
FILENAME_FOR_STUDENTS_IMPORT: students_export.txt
```

Is it about the spaces or quotes?

Reply:
This is some YML syntax that lets us put the value on a newline instead of directly after the key, I did that because some of the values are reaaaaally long and much easier to read if they're on their own line.